ui-components/src/forms/Field.tsx

```diff
@@ -41,7 +43,20 @@ const Field = (props: FieldProps) => {
     controlClasses.push(props.controlClassName)
   }
 
-  const type = props.type || "text"
+  const formatValue = () => {
+    if (props.getValues && props.setValue) {
+      const currencyValue = props.getValues()[props.name]
```

Comment:
I think it's okay, but I suppose simpler would be: `props.getValues(props.name)`